### ISSUE TITLE
Docker image not renaming files after converting

### ISSUE THREAD

**Issue by samcro1967** [mdhiggins/sickbeard_mp4_automator#1064]
Not having any luck getting the docker sma images working completely.  Sonarr downloads and moves the episode to the download folder, moves it to the TV folder, converts it to mp4, but, never gets renamed.  The /var/logs/sickbeard_mp4_automator/index.log is empty.  Not sure where to look for more clues.

**Container creation command**
docker create --name=sonarr-sma -p 8989:8989 -v C:\Users\username\Documents\Docker\sonarr:/config -v C:\Users\username\Downloads\complete\tv:/downloads -v C:\Users\username\Documents\Docker\sma\autoProcess.ini:/usr/local/bin/sma/sickbeard_mp4_automator/autoProcess.ini -v //K/xbmc/TV:/tv mdhiggins/sonarr-sma

Changed Connections custom script path and argument to /usr/local/bin/sma/env/bin/python3 & /usr/local/bin/sma/sickbeard_mp4_automator/postSonarr.py.  It works when tested.

Here is the Sonarr log.
[Sonarr.txt](https://github.com/mdhiggins/sickbeard_mp4_automator/files/3665560/Sonarr.txt)

Here is the sabnzbd log.
[sabnzdb.txt](https://github.com/mdhiggins/sickbeard_mp4_automator/files/3665573/sabnzdb.txt)

**Comment by mdhiggins** [comment]
Turn on completed download handling

**Comment by samcro1967** [comment]
Enabled, but still getting the same result.

**Comment by mdhiggins** [comment]
so this is a sonarr problem then. If renaming isn’t occurring then it’s not getting passed to sonarr. The script isn’t even in play yet so I would check your sonarr configuration and paths.

**Comment by samcro1967** [comment]
Appears -v //K/xbmc/TV:/tv does not work for a CIFS share.  Docker does not throw an error when creating the container, but the directory in the container is empty.  Enabled NFS on the NAS and created an NFS volume.  Attached it to the container and get the same results.  The mount is there, but the folder is blank.  Can;t seem to get the container to see remove volumes.

**Comment by samcro1967** [comment]
I was able to get this working by creating the volume using the following syntax:

docker volume create --driver local --opt type=cifs --opt device=//nasdevice/share/ --opt o=username=user1,password=user1password networkkdrive

I then created the container with this syntax:

docker create --name=sonarr-sma -p 8989:8989 -e PUID=0 -e PGID=0 -v C:\Docker\sonarr:/config -v C:\Downloads\complete\tv:/downloads -v C:\Docker\sma\autoProcess.ini:/usr/local/bin/sma/sickbeard_mp4_automator/autoProcess.ini -v networkdrive:/tv mdhiggins/sonarr-sma

I am then able to connect to the container using:

docker exec -t sonarr-sma bash

From there I am able to copy and delete files so I know I have read/write access from inside the container to the network drive.  

I then see /tv/ as the default path when I add a new series in Sonarr.

When I go to series editor, select all show, open the root folder drop down box, select /tv/, and click save, it changes the path for all series from "k:\" to "/tv/k:\".  I think this is an issue on the Sonarr side?  If so, I will submit an issue there.
